dali/pipeline/operators/resize/random_resized_crop.cc

```diff
@@ -29,15 +29,20 @@ DALI_SCHEMA(RandomResizedCrop)
   .NumOutput(1)
   .AllowMultipleInputSets()
   .AddOptionalArg("random_aspect_ratio",
-      R"code(Range from which to choose random aspect ratio.)code",
+      R"code(Range from which to choose random aspect ratio (width/height).)code",
```

Comment:
can you also update the documentation in host_decoder_random_crop.cc?

Reply:
Done.